shared/teams/new-team/wizard/add-subteam-members.tsx

```diff
@@ -0,0 +1,144 @@
+    state => Constants.getTeamDetails(state, parentTeamID).members
+  )
+  const parentMembers = [...parentMembersMap.values()].filter(m => !Constants.isBot(m.type))
+  const filteredMembers = filter
+    ? parentMembers.filter(
+        m => m.username.toLowerCase().includes(filterL) || m.fullName.toLowerCase().includes(filterL)
+      )
+    : parentMembers
+  const allSelected = parentMembers.length === selectedMembers.size
+  const onSelectAll = () => setSelectedMembers(new Set(parentMembers.map(m => m.username)))
+  const onSelectNone = () => setSelectedMembers(new Set())
+
+  const continueLabel = selectedMembers.size
+    ? `Continue with ${selectedMembers.size} ${pluralize('member', selectedMembers.size)}`
+    : 'Continue without members'
+  const doneLabel = selectedMembers.size ? 'Done' : 'Skip'
+
+  const renderItem = (_: number, m: Types.MemberInfo) => (
+    <Kb.ListItem2
+      type="Small"
+      icon={<Kb.Avatar username={m.username} size={32} />}
+      body={
+        <Kb.Box2 direction="vertical">
+          <Kb.ConnectedUsernames type="BodySemibold" usernames={[m.username]} />
+          <Kb.Text type="BodySmall">{m.fullName}</Kb.Text>
+        </Kb.Box2>
+      }
+      action={
+        <Kb.CheckCircle
+          checked={selectedMembers.has(m.username)}
+          onCheck={check => {
+            // TODO: ensure performance
+            check ? selectedMembers.add(m.username) : selectedMembers.delete(m.username)
+            setSelectedMembers(new Set([...selectedMembers]))
+          }}
+        />
+      }
+      firstItem={true}
+      key={m.username}
+    />
+  )
+  return (
+    <Kb.Modal
+      allowOverflow={true}
+      mode="DefaultFullHeight"
+      onClose={onClose}
+      header={{
+        leftButton: <Kb.Icon type="iconfont-arrow-left" onClick={onBack} />,
+        rightButton: Styles.isMobile ? (
+          <Kb.Text type="BodyBigLink" onClick={onContinue}>
+            {doneLabel}
+          </Kb.Text>
+        ) : (
+          undefined
+        ),
+        title: <ModalTitle teamID={Types.newTeamWizardTeamID} title="Add members" />,
+      }}
+      footer={
+        Styles.isMobile
+          ? undefined
+          : {
+              content: <Kb.Button label={continueLabel} onClick={onContinue} fullWidth={true} />,
+            }
+      }
+      noScrollView={true}
+    >
+      <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.hideOverflow}>
+        <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.searchContainer}>
+          <Kb.SearchFilter
+            size="full-width"
+            onChange={setFilter}
+            value={filter}
+            placeholderText={`Search ${parentMembers.length} members`}
+            icon="iconfont-search"
+            style={styles.search}
+          />
+        </Kb.Box2>
+        {/* TODO: once it's easier to make a single different-height header, make this part of the list2 */}
```

Comment:
I don't think it's bad this doesn't scroll. But agree would be nice for this to be easier.

Reply:
I asked Cècile and she said it should scroll but I'd already implemented it this way by the time she replied so I added this comment.